compiler/docgen.nim

```diff
@@ -1357,6 +1434,7 @@ proc commandRst2TeX*(cache: IdentCache, conf: ConfigRef) =
   commandRstAux(cache, conf, conf.projectFull, TexExt)
 
 proc commandJson*(cache: IdentCache, conf: ConfigRef) =
+  ## implementation of a deprecated jsondoc0 command
```

Comment:
then should we rename commandJson as commandJson0 ? (can be done in future PR)

Reply:
I thought about deleting this `jsondoc0` altogether. It always crashes for me, including stable 1.4 version. Does it work for you?

Reply:
crashes with `options.nim(637, 3) `not conf.outFile.isEmpty`  [AssertionDefect]` but that doesn't mean it'd be hard to fix; if this were to be removed, it should be a separate PR though (IIRC the use case is for include files where semcheck don't work; just like nim doc0; it's hacky but ./koch docs relies on at least nim doc0)

Reply:
may be but i'm not planning on doing any fixes, because i don't understand the use cases of `jsondoc/jsondoc0` commands in the 1st place :-/

Reply:
`nim jsondoc` use case is for tooling, so that a program can use the json output generated by nim for further processing (eg to support queries, such as find all procs matching a given signature) without having to link against nim or use compiler API, refs https://github.com/nim-lang/Nim/issues?q=is%3Aissue+is%3Aopen+jsondoc.

But yes, defering to future work is good